stdlib/2and3/lib2to3/pgen2/tokenize.pyi

```diff
@@ -0,0 +1,30 @@
+# Stubs for lib2to3.pgen2.tokenize (Python 3.6)
+# NOTE: Only elements from __all__ are present.
```

Comment:
Untokenizer and a few others actually aren't in `__all__`. There's a lot more public names in this file, but probably not worth it to add them all.